File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    
    s = s.lower()
    countP = 0
    countY = 0
    
    for alph in s:
        if alph == 'p':
            countP +=1
        elif alph == 'y':
            countY +=1
            
    if countP != countY:
        answer = False
    
    return answer

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
import java.util.*;

class Solution {
    boolean solution(String s) {
        boolean answer = true;

        // s 모두 소문자로 변경
        String sen = s.toLowerCase();
        // s에 'p'의 개수와 'y'의 개수를 비교
        int p = 0;
        int y = 0;
        for(int i = 0; i < sen.length(); i++){
            if(sen.charAt(i) == 'p'){
                p+=1;
            }
            if(sen.charAt(i) == 'y'){
                y+=1;
            }
        }
            
        // 같으면 true, 다르면 false,
        if(p != y){
            answer = false;
        }

        return answer;
    }
}